exercises/pig-latin/.meta/gen.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"log"
+	"text/template"
+
+	"../../../gen"
+)
+
+func main() {
+	t, err := template.New("").Parse(tmpl)
+	if err != nil {
+		log.Fatal(err)
+	}
+	var j js
+	if err := gen.Gen("pig-latin", &j, t); err != nil {
+		log.Fatal(err)
+	}
+}
+
+// The JSON structure we expect to be able to unmarshal into
+type js struct {
+	Exercise string
+	Version  string
+	Cases    []oneLevel
+}
+
+// Test levels
+type oneLevel struct {
+  Description string
+  Cases []oneCase
+}
+
+// Test cases
+type oneCase struct {
+	Description string
+	Property    string
+	Input       string
+	Expected    string
+}
+
+// Template to generate test cases.
+var tmpl = `package igpay
```

Comment:
I know it was there before but what does it mean ? IMHO, better to make it `piglatin` and rename the functions to `Sentence` and `Word`. what do you think ?